manticore/core/smtlib/solver.py

```diff
@@ -39,15 +52,26 @@
     default=10000,
     description="Maximum solutions to provide when solving for all values",
 )
-consts.add("z3_bin", default="z3", description="Z3 binary to use")
+consts.add("z3_bin", default="z3", description="Z3 solver binary to use")
+consts.add("cvc4_bin", default="cvc4", description="CVC4 solver binary to use")
+consts.add("yices_bin", default="yices-smt2", description="Yices solver binary to use")
+
+
 consts.add("defaultunsat", default=True, description="Consider solver timeouts as unsat core")
 consts.add(
     "optimize", default=True, description="Use smtlib command optimize to find min/max if available"
 )
 
+consts.add(
+    "solver",
+    default=SolverType.yices,
```

Comment:
The README says that Z3 is still the default. Which do we want to use? I'm in favor of leaving Z3 as the default since we have a long history of testing against it and it's very easy to install.

Reply:
I set it back to Z3.  (I hope it passes tests. :joy: )